docs/index.rst

```diff
@@ -7,6 +7,39 @@
 Welcome to bodhi's documentation!
 =================================
 
+What is Bodhi?
+--------------
+Bodhi is a modular web-based system that facilitates the process of publishing package updates for Fedora. Bodhi is the application that manages updates for Fedora releases. It maintains a single stage of repositories by adding/updating/removing packages. It can be used to create or modify updates and overrides.
+
+Advantages of bodhi
+-------------------
+
+1.  Provides an intuitive interface for developers and release engineers to manage pushing out package updates for multiple version releases.
+2.  Helps in delivering quality packages and repository sustainment with automated testing.
+3.  Helps Community testers to get involved in testing of the package updates and provide quality feedback which would reach the packagers/developers.
```

Comment:
I'd probably reword the above a bit. Something like:

"Gives community testers an interface to leave feedback about package updates."